CDCgov/prime-reportstream
# Use test organizations.yml file in translation tests

This PR points to a dedicated test `organizations.yml` file for translation tests in `TranslationTests.kt`. As part of this test, I updated the test organization settings to include some translation settings from staging and prod for ELIMS and SR/CA and I updated the appropriate translation tests to make use of those settings.

Test Steps:
1. Run the translation tests.
2. Adjust settings in `prime-router/settings/organizations.yml` and rerun the tests to see that the behavior does not change
3. Adjust settings in `prime-router/src/testIntegration/resources/settings/organizations.yml` and rerun the tests to see that the behavior changes

## Changes
- Adjust the settings directory specified in `TranslationTests.kt`
- Add additional test organization settings to the test organizations file
- Adjust the expected output for SR/CA translation tests based on the updated settings
- Allow sender transform settings to be obtained from the sender settings if the sender is set but the sender schema is not set
- Allow receiver transform settings to be obtained from the receiver settings if the receiver is set but the output schema is not set

## Checklist

### Testing

## Linked Issues
- Addresses #11416
